pkg/driver/aci_driver.go

```diff
@@ -0,0 +1,770 @@
+package driver
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/authorization/mgmt/2015-07-01/authorization"
+	"github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2018-10-01/containerinstance"
+	"github.com/Azure/azure-sdk-for-go/services/msi/mgmt/2018-11-30/msi"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2015-11-01/subscriptions"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/google/uuid"
+
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+const userAgent string = "Duffle ACI Driver"
+const msiTokenEndpoint = "http://169.254.169.254/metadata/identity/oauth2/token"
+
+// ACIDriver runs Docker and OCI invocation images in ACI
+type ACIDriver struct {
+	config map[string]string
+	// This property is set to true if Duffle is running in cloud shell
+	inCloudShell bool
+	// If true, this will not actually create ACI instance
+	Simulate           bool
```

Comment:
Why is this field exported ? I don't see where it is used outside this package 🤔